text-ui-test/EXPECTED.TXT

```diff
@@ -14,7 +14,12 @@
 Error Message:
     Invalid command issued!
 Note:
-    Please try 'help' command to check the available commands and their usages!
+    Access Atomic Habit: hb
+    Access Self Reflection : reflect
+    Access Focus Timer: ft
+    Access Gamification: gamif
+    Help command: help
+    Exit program: exit
```

Comment:
If you implement my above suggestions about `MainManager`, might need to update this again